gremlin-go/driver/graphBinary.go

```diff
@@ -718,6 +718,8 @@ func (serializer *graphBinaryTypeSerializer) getType(val interface{}) (dataType,
 			// We can write an array or slice into the list dataType.
 			return listType, nil
 		default:
+			fmt.Println("DSL??")
```

Comment:
Note: Don't forget to remove this.

Reply:
@simonz-bq Someone may want to know `DSL??` though...

Reply:
LOL, yes sorry that was for debug, will remove